songs/farin-urlaub-racing-team/die-perfekte-diktatur.tex

```diff
@@ -0,0 +1,92 @@
+\beginsong{Die perfekte Diktatur}[by={Farin Urlaub Racing Team}]
+
+    \textnote{Your guitar should be tuned to an E$\,\flat$}
+    Intro: Am Em Fmaj7 Fadd9 \rep{2}
+    \beginverse
+        \[Am] Ich geh nicht mehr ans Tele\[Em]fon,
+        es klingelt jeden Tag um \[Fmaj7]vier. \[Fadd9]
+        \[Am] Ich fahr nicht mit der Strassen\[Em]bahn,
+        ich lauf den ganzen Weg zu \[Fmaj7]dir. \[Fadd9]
+        \[C] Das Telefon wird abge\[G]hört
+        und in der Bahn sitzen Spi\[Dm]one.
+        \[Am] Sie tragen stets dieselbe \[Em]Uhr
+        und wissen alle wo ich \[Fmaj7]wohne.
+        \[Am]Wir treffen uns in einem \[Em]Park,
+        um diese Zeit ist niemand \[Fmaj7]hier. \[Fadd9]
+        \[Am] Wir können reden unge\[Em]stört
+        und es gibt keine Mikro\[Fmaj7]fone. \[Fadd9]
+    \endverse
+
+    \beginchorus
+      \[Dm]Sie kommen meistens nachts um die Leute zu \[Am]holen,
+      und hinterher steht \[C]wieder eine Wohnung leer. \[G]
+      Die Zeitungen sind \[Dm]voll mit Durchhaltepa\[Am]rolen,
+      aber Zeitung \[C]liest sowieso keiner mehr. \[G]
+      Überall sind \[Dm]Kameras angebracht,\[Am]
+      alles wird \[C]rund um die Uhr überwacht. \[G]
+    \endchorus
+
+    \beginchorus
+      Die Grenzen sind ge\[Dm]schlossen seit Jahren schon,\[E]
+      dir bleibt nur noch die \[Am]innere \[G]Emigra\[Am]tion.
+    \endchorus
+
+    \beginverse
+    \[Am] Ich kenn dich seit wir Kinder \[Em]sind,
+    ich weiß ich kann dir auch ver\[Fmaj7]trau'n \[Fadd9]
+    \[Am] Wir planen schon seit einem Jahr \[Em]
+    zusammen von hier abzu\[Fmaj7]hau'n. \[Fadd9]
+    \[C]Du hast uns Dynamit bes\[G]orgt,
+    jetzt brauchen wir noch ein paar \[F]Waffen. \[Dm]
+    \[Am] Du sagst du hälst es nicht mehr \[Em]aus
+    und du hoffst sehr dass wir es \[Fmaj7]schaffen. \[Fadd9]
+    \[Am] Zum Abschied darf ich dir noch ein\[Em]mal
+    tief in deine Augen schaun\[Fmaj7]. \[Fadd9]
+    \[Am] Ich denk den ganzen Weg nach \[Em]Haus an dich,
+    ich werd wohl nicht viel \[Fmaj7]schlafen. \[Fadd9]
+    \endverse
+
+    \beginchorus
```

Comment:
Since you reference a **chorus** on the same page: Is this the chorus that has to be replayed (there are 2 defined)?
If so I'd add a marking like this:
```suggestion
    \beginchorus
       \textbf{Chorus}
```
People then can reference this immediately.